go.mod

```diff
@@ -28,8 +28,8 @@ require (
 	github.com/thoas/go-funk v0.8.0
 	golang.org/x/lint v0.0.0-20210508222113-6edffad5e616 // indirect
 	golang.org/x/mod v0.4.2
-	golang.org/x/sys v0.0.0-20210521090106-6ca3eb03dfc2 // indirect
-	golang.org/x/tools v0.1.1
```

Comment:
There should be no module changes in this PR.

Reply:
Modified.